src/components/AppModal.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Modal } from '@mui/material'
+import { SxProps } from '@mui/system'
+import React from 'react'
+
+type Props = {
+  isOpen: boolean
+  onClose: () => void
+  children: React.ReactElement
+  style?: SxProps
+}
+
+const defaultStyle: SxProps = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  minWidth: 400,
+  bgcolor: 'var(--col-light)',
+  boxShadow: 24,
+  p: 4,
+  borderRadius: '8px',
+}
+
+const AppModal = ({ isOpen, onClose, children, style = defaultStyle }: Props) => {
```

Comment:
if you override `style` prop, all the default styles will disappear

let's better use `sx={{ ...defaultStyle, ...style }}` or even `sx` instead of `style`